src/sql/workbench/services/notebook/browser/models/notebookModel.ts

```diff
@@ -963,12 +963,20 @@ export class NotebookModel extends Disposable implements INotebookModel {
 
 	private async updateKernelInfoOnKernelChange(kernel: nb.IKernel, kernelAlias?: string) {
 		await this.updateKernelInfo(kernel);
+		let aliasLanguageInfo: nb.ILanguageInfo;
 		this.kernelAliases.forEach(kernel => {
 			if (this._defaultLanguageInfo?.name === kernel.toLowerCase()) {
 				kernelAlias = kernel;
 			}
 		});
-		if (kernel.info) {
+		if (kernelAlias) {
+			aliasLanguageInfo = {
```

Comment:
Since aliasLanguageInfo isn't used anywhere else, can we just declare it here:
`let aliasLanguageInfo: nb.ILanguageInfo = {`